### PR TITLE
add 2.0.2 and 2.0.3 version in the issue-bug-report

### PR DESCRIPTION
**Pull request by zhuangchong**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -98,6 +98,8 @@ body:
         Which version of Apache DolphinScheduler are you running? We only accept bugs report from the LTS projects.
       options:
         - dev
+        - 2.0.3
```
Comment (zhongjiajie) [inline .github/ISSUE_TEMPLATE/bug-report.yml:101]: it seem we do not release version 2.0.3. it there any reason you want to add it?

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (zhuangchong) [reply .github/ISSUE_TEMPLATE/bug-report.yml:101]: It is a bug test requirement before 2.0.3 release.

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**